Dockerfile

```diff
@@ -10,7 +10,7 @@ COPY ./requirements.txt /app/requirements.txt
 RUN apt-get -q update \
     && apt-get -q --yes install g++ \
     && pip install --upgrade --no-cache-dir -r requirements.txt \
-         gunicorn nose flake8 \
+         gunicorn nose flake8 mock webtest unittest2 \
```

Comment:
Just out of curiosity, why do we want dev deps like nose or mock in the Docker instance? (A comment would be valuable IMO)